tfe/data_source_organization_tags_test.go

```diff
@@ -0,0 +1,60 @@
+package tfe
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccTFEOrganizationTagsDataSource_basic(t *testing.T) {
+	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+	orgName := fmt.Sprintf("tst-terraform-%d", rInt)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTFEOrganizationTagsDataSourceConfig(rInt),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"data.tfe_organization_tags.foobar", "organization", orgName),
+					resource.TestCheckResourceAttrSet("data.tfe_organization_tags.foobar", "tags"),
+				),
+			},
+		},
+	})
+}
+
+func testAccTFEOrganizationTagsDataSourceConfig(rInt int) string {
+	return fmt.Sprintf(`
+	resource "tfe_organization" "foobar" {
+		name  = "tst-terraform-%d"
+		email = "admin@company.com"
+	  }
```

Comment:
The indentation on this terraform config string is kinda messed up, probably because your editor was trying to "help" by adding Go-appropriate tab literals. Could I get you to de-indent the top-level of this config so it's flush against the left, and then use two spaces for indenting block bodies?

This is lower priority than my other comments.